release-notes.md

```diff
@@ -1,3 +1,19 @@
+# Scorpius-19.10.0
+## Add permission to block new direct login users ##
```

Comment:
i would rephrase it to Add permission to allow setting SSO per user
like the permission name you gave